tox.ini

```diff
@@ -567,7 +567,7 @@ basepython=python
 # DEV: We use `conftest.py` as a local pytest plugin to configure hooks for collection
 [pytest]
 # Common directories to ignore
-addopts = --ignore "tests/utils" --ignore "tests/base" --benchmark-disable
+addopts = --ignore "tests/utils" --ignore "tests/base"
```

Comment:
This will run the benchmark for everything in the CI which can take sometimes so I don't think it's a good idea. ;)

Reply:
Ahaha, I forgot to remove this change! 😆